content/docs/reference/pack/pack.md

```diff
@@ -0,0 +1,41 @@
++++
+title="pack"
++++
+## pack
+
+CLI for building apps using Cloud Native Buildpacks
+
+### Synopsis
+
+CLI for building apps using Cloud Native Buildpacks
```

Comment:
Why do all the Synopsis sections repeat the description?

Reply:
It looks like Synopsis prints out the `cmd.Long` description, and that just combines the `cmd.Short` with the usage if there is no separate long description.